test/e2e/autonomy/autonomy_test.go

```diff
@@ -258,7 +258,7 @@ var _ = ginkgo.AfterSuite(func() {
 	ginkgo.By("delete namespace:" + YurtE2ENamespaceName)
 	gomega.Eventually(func() error {
 		return ns.DeleteNameSpace(c, YurtE2ENamespaceName)
-	}).WithTimeout(10 * time.Second).WithPolling(1 * time.Second).Should(gomega.Succeed())
+	}).WithTimeout(20 * time.Second).WithPolling(1 * time.Second).Should(gomega.Succeed())
```

Comment:
It seems that ns.DeleteNameSpace already has its own timeout which is 5min. Timeout we set here may have conflict with the original timeout.

Could we split the timeout as `delete ns timeout` and `reconnect timeout`, and only set `reconnect timeout` here?

I think we can use something like `clientset.Discovery().ServerVersion()` to detect if APIServer has reconnected.